src/main/resources/db/migration/V1.9.0__Replace_empty_value_to_null.sql

```diff
@@ -0,0 +1,3 @@
+use seichiassist;
+
+UPDATE playerdata SET shareinv = NULL WHERE shareinv = '';
```

Comment:
```suggestion
UPDATE playerdata SET shareinv = NULL WHERE shareinv = '';

```

nit: POSIXでは、ファイルの最後の文字がキャレッジリターンではない場合テキストとしてみなされないので最後にキャレッジリターンを付与してください。